hasktorch/src/Torch/Script.hs

```diff
@@ -243,7 +247,7 @@ instance Parameterized ScriptModule where
     return $ updateParameters WithRequiredGrad module' (map toDependent ps')
 
 trace :: String -> String -> ([Tensor] -> IO [Tensor]) -> [Tensor] -> IO RawModule
-trace moduleName functionName func inputs = cast3 (\m f inps -> LibTorch.trace m f (trans func) inps) moduleName functionName inputs
+trace moduleName functionName func = cast3 (\m f inps -> LibTorch.trace m f (trans func) inps) moduleName functionName
```

Comment:
in this module are some eta-reduced function bodies.

Reply:
this here is just an example.
Could you see please if you can document some of the arguments here, as before just by name?